lca-chimevc-stack/lambda_functions/chime_call_analytics_initialization/index.js

```diff
@@ -623,11 +626,15 @@ const handler = async function handler(event, context) {
         );
         return;
       }
+      
+      let tempCallData = await getCallDataFromChimeEvents(event);
+      callData = await getCallDataFromDdb(tempCallData.callId);
 
-      callData = await getCallDataFromChimeEvents(event);
+      console.log('Modified callData:');
```

Comment:
why log this as 'Modified callData' ?   It doesn't seem modified..  Can you clarify the log message?